### ISSUE TITLE
Section headings in Sanity call outs are not styled as heading tags

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#1358]
In some blogs, content that looks like it should be a heading is not. I think that this is because they are in call outs (or similar elements) and aren't being styled as headings. 

https://owa.thenational.academy/blog/how-to-provide-effective-feedback "Tell us how you got on" should be a heading

**Acceptance Criteria:**
- Ensure that elements that are labelled as "Section headings" in sanity keep the heading property as they are passed through to the Front end

**Comment by jack-skerrett-oak** [comment]
https://www.notion.so/oaknationalacademy/Within-some-of-the-blog-articles-there-are-text-elements-that-are-visually-styled-as-headings-but-th-b1f58850b5884ab7bce417e03c9a8d3d?pvs=4

**Comment by jack-skerrett-oak** [comment]
moved to notion
